.eslintrc.json

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": ["react-app", "plugin:@typescript-eslint/recommended"],
```

Comment:
Don't know about you, but I'm a fan of Prettier and usually add it as a plugin in `eslint` config.

Reply:
We will add them later after this.